contracts/universalResolver/UniversalResolver.sol

```diff
@@ -204,74 +203,79 @@ contract UniversalResolver is ERC165, Ownable {
     }
 
     function reverse(
-        bytes calldata reverseName
-    ) external view returns (string memory, address, address, address) {
-        return reverse(reverseName, batchGatewayURLs);
+        bytes calldata lookupAddress,
+        uint256 coinType
+    ) external view returns (string memory, address, address) {
+        return reverse(lookupAddress, coinType, batchGatewayURLs);
     }
 
     /**
      * @dev Performs ENS name reverse resolution for the supplied reverse name.
-     * @param reverseName The reverse name to resolve, in normalised and DNS-encoded form. e.g. b6E040C9ECAaE172a89bD561c5F73e1C48d28cd9.addr.reverse
-     * @return The resolved name, the resolved address, the reverse resolver address, and the resolver address.
+     * @param lookupAddress The reverse address to resolve, eg. 0xb6E040C9ECAaE172a89bD561c5F73e1C48d28cd9
+     * @return The resolved name, the resolver address, and the reverse resolver address.
      */
     function reverse(
-        bytes calldata reverseName,
+        bytes memory lookupAddress,
+        uint256 coinType,
         string[] memory gateways
-    ) public view returns (string memory, address, address, address) {
-        bytes memory encodedCall = abi.encodeCall(
-            INameResolver.name,
-            reverseName.namehash(0)
+    ) public view returns (string memory, address, address) {
+        require(coinType == 60, "reverse: expected coinType 60");
```

Comment:
Can you open an issue in Linear to track adding support for reverse lookups of other cointypes to the universal resolver?